habitat_baselines/rl/ddppo/policy/resnet_policy_bpsnav.py

```diff
@@ -321,16 +348,29 @@ def __init__(
         if discrete_actions:
             self.prev_action_embedding = nn.Embedding(action_space.n + 1, 32)
         else:
-            assert(len(action_space.shape) == 1)
+            assert len(action_space.shape) == 1
             self.prev_action_embedding = nn.Linear(action_space.shape[0], 32)
 
         self._n_prev_action = 32
         rnn_input_size = self._n_prev_action
 
-        if "pointgoal_with_gps_compass" in observation_space.spaces:
-            n_input_goal = 3
-            self.tgt_embeding = nn.Linear(n_input_goal, 32)
-            rnn_input_size += 32
+        self._include_inputs: List[str] = [
+            k
+            for k in include_inputs
```

Comment:
```suggestion
            for k in observation_space.spaces
```
We already have a list of sensors we want to include in gala_kinematic_ddppo.yaml